traveltimepy/sdk.py

```diff
@@ -670,10 +704,8 @@ def get_time_info(departure_time: Optional[datetime], arrival_time: Optional[dat
     if departure_time and arrival_time:
         raise ApiError("arrival_time and departure_time cannot be both specified")
 
-    time_info = None
     if departure_time:
-        time_info = DepartureTime(departure_time)
+        return DepartureTime(departure_time)
     elif arrival_time:
-        time_info = ArrivalTime(arrival_time)
+        return ArrivalTime(arrival_time)
 
```

Comment:
simplified this method, not sure why I did it in such an interesting way last time. 

We should rework the arrival/departure times more heavily like we talked some time ago, to prevent mixups (we didn't do it because we wanted to avoid breaking changes).